celery/backends/arangodb.py

```diff
@@ -60,7 +60,7 @@ def __init__(self, url=None, *args, **kwargs):
                 database_collection, _query
             ) = _parse_url(url)
             logging.debug(
```

Comment:
Why do we need this log line?

Reply:
Its just the configuration of the backend.
Are there any strict guidelines regarding the logs as I'm even logging on a get, set, mget and delete